vue-app/src/api/user.ts

```diff
@@ -55,3 +60,36 @@ export async function getEtherBalance(
 ): Promise<BigNumber> {
   return await provider.getBalance(walletAddress)
 }
+
+export async function getBrightId(contextId: string): Promise<BrightId> {
+  const brightId: BrightId = {
+    isLinked: false,
+    isSponsored: false,
+    isVerified: false,
+  }
+
+  try {
+    await getVerification(contextId)
```

Comment:
So we don't do anything with this return value? Only if the request errors?

Reply:
(that's fine - I see we use it elsewhere when we actually need to create the tx to add user to the registry)

Reply:
Hmm that's a good point. The thing is that if the query went well, that means that the user is verified and we receive a verification sig.
Perhaps the only thing we could check is the `unique` field that is returned in the payload after a successful verification. https://dev.brightid.org/docs/node-api/web_services/foxx/node-api-5.6.0.yaml/paths/~1verifications~1%7Bapp%7D~1%7BcontextId%7D/get
I'm not entirely sure when that field could return false 🤔 and if that field should be used as our `isVerified` field.

Reply:
Will include it in the next PR.